### PR TITLE
Improve Jest ESM config

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

Pretty straightforward: the long regex made git merge conflicts harder to deal with and it was unreadable. This turns it into a regular array

## Checklist

- [x] Designate a primary reviewer: @grahamlangford

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -15,6 +15,41 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const esmPackages = [
+  "@cfworker|escape-string-regex",
```
Comment (twschiller) [inline jest.config.js:19]: Is this correct? Should it be split into 2 lines?

**Review by twschiller — APPROVED** [review]
See comment: https://github.com/pixiebrix/pixiebrix-extension/pull/7864/files#r1518565220

File: jest.config.js (hunk 1 above)
Comment (twschiller) [reply jest.config.js:19]: Also, I'd recommend sorting the lines?

Comment (fregante) [reply jest.config.js:19]: Indeed it's not correct, but jest passes so they can both be dropped 🙌

**Pull request merged**